src/Api/RequestAttributesExtractor.php

```diff
@@ -29,19 +31,17 @@
      *
      * @throws RuntimeException
      *
-     * @return array
+     * @return array [$resourceClass, $collectionOperation, $itemOperation, $format]
```

Comment:
IIUC `@example` is used to import external files as examples: https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.example.pkg.html

According to http://stackoverflow.com/questions/15414103/best-way-to-document-array-options-in-phpdoc there is not better way that what I've done here but I'm to change that if someone has something better :)